src/controllers/enterpriseController.test.ts

```
@@ -93,17 +95,129 @@ describe("enterpriseController", () => {
+    await getTreasury(
+      { apiKey: { userId: "user-1", organizationId: "org-1", permissions: ["enterprise:read"], rateLimit: 100 }, query: {} } as unknown as AuthRequest,
+      res,
+      next,
+    );
+
+    expect(next).toHaveBeenCalledWith(expect.any(Error));
+  });
+
+  it("handles missing organizationId gracefully", async () => {
+    mockGetTreasury().mockResolvedValue({
+      totalBalanceUsd: 0, totalReserveAmount: 0,
```

ESLint

[error] Insert `⏎·····`

(prettier/prettier)